src/main/kotlin/uk/gov/justice/digital/hmpps/prisonercellallocationapi/model/CellMovement.kt

```diff
@@ -14,11 +16,17 @@ data class CellMovement(
   @GeneratedValue(strategy = GenerationType.IDENTITY)
   val id: Long? = null,
   val agency: String,
-  val cellId: Long? = null,
-  val cellDescription: String? = null,
+  val cellId: Long,
+  val cellDescription: String,
   val prisonerId: String,
   val prisonerName: String,
   val userId: String,
   val dateTime: LocalDateTime,
   val reason: String,
+  @Enumerated(EnumType.STRING)
+  val moveType: MoveType,
 )
+
+enum class MoveType {
```

Comment:
Can we call this DIRECTION? I imagine at some point in the future we'll have a move type eg general moves, safety (similar to reason)